tests/numerics/project_parallel_q_03.with_trilinos=true.mpirun=1.output

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
I think that it would be enough to only run the test `project_parallel_q_01` with both 1 and 4 processors and the `02` through `05` tests only with 4 processors. Even though we like to have as many test as possible, running the exact same thing 20 times (debug, release, 5 test variants that only differ in some aspects of the mesh) seems a bit too much.